src/app/pages/MapPage/Loadable.tsx

```diff
@@ -0,0 +1,11 @@
+/**
+ * Asynchronously loads the component for MapPage
+ */
```

Comment:
This doc should accompany every component function to describe what the component is. You don't have to be too detailed but it is good to explain what header is when it is rendered